outlines/1.3-migration

```diff
@@ -0,0 +1,31 @@
+1. Breaking changes (as we'd normally do this)
+
+2. Recommended changes : Modules [link to app structure guide]
+  1. Applications: Move all files to imports, initialize in one spot, link via `import`
+  2. Import *all* symbols from packages, including meteor core packages.
```

Comment:
> Speaking of which, do you currently still get globals from Meteor packages even when your file is in imports?

I would have thought so, I mean Meteor puts those symbols on `window`; it doesn't matter where the code being executed lives.

>  I wonder how hard it would really be to improve the APIs of all the core packages

Yeah I mean if we are going to recommend that package authors `export const X` from their packages, then we should really do the same, even if we leave the `api.exports()` (or attaching to the `Meteor` namespace) for backwards compat.

We should probably call stuff out in the docs as deprecated too. I fear to ask, but is there a way to do this ;)